docs/en-us/cloneset.md

```diff
@@ -107,8 +107,60 @@ when controller receives above update request, it ensures the number of replicas
 deleted, the Pods listed in `podsToDelete` will be deleted first.
 Controller will clear `podsToDelete` automatically once the listed Pods are deleted. Note that:
 
-- If one just adds a Pod name to `podsToDelete` and do not modify `replicas`, controller will delete this Pod, and create a new Pod.
-- Without specifying `podsToDelete`, controller will scale down by deleting Pods in the order: not-ready < ready, unscheduled < scheduled, and pending < running.
+If one just adds a Pod name to `podsToDelete` and do not modify `replicas`, controller will delete this Pod, and create a new Pod.
+For the migration condition, an easier way is to add a label `apps.kruise.io/specified-delete: true` onto the Pod waiting to delete.
+
+Comparing to delete the Pod directly, using `podsToDelete` or `apps.kruise.io/specified-delete: true` to let CloneSet delete and recreate is better.
```

Comment:
i think we should be more neutral to the native deletion and specified-delete. native deletion will not reduce the application replicas, yet one can scale down the replicas with specified deletion.